### ISSUE TITLE
Pageable - Security issue

### ISSUE THREAD

**Issue by federicojasson** [nicogarcia/classes-web-server#68]
Pageable permite por defecto ordenar las búsquedas por cualquier propiedad del objeto.
	
¿Puede ser esto un riesgo de seguridad?
Las siguientes consultas funcionan:

/teachers?sort=user.password,ASC
/teachers?sort=user.password,DESC

**Comment by nicogarcia** [comment]
¡Holy shit! No creo que les pueda servir de algo, son los hashes en binario.

**Comment by federicojasson** [comment]
Yep, aunque también se podría usar para los planes y demás. Lo dejo abierto para tenerlo en cuenta en el futuro.
